builtins-default-costs/src/lib.rs

```diff
@@ -5,36 +5,125 @@ use {
     lazy_static::lazy_static,
     solana_sdk::{
         address_lookup_table, bpf_loader, bpf_loader_deprecated, bpf_loader_upgradeable,
-        compute_budget, ed25519_program, loader_v4, pubkey::Pubkey, secp256k1_program,
+        compute_budget, ed25519_program,
+        feature_set::{self, FeatureSet},
+        loader_v4,
+        pubkey::Pubkey,
+        secp256k1_program,
     },
 };
 
+/// DEVELOPER: when a builtin is migrated to sbpf, please add its corresponding
+/// migration feature ID to BUILTIN_INSTRUCTION_COSTS, so the builtin's default
+/// cost can be determined properly based on feature status.
```

Comment:
100% better not to maintain two separate lists. Concerning backport, wdyt implement your suggestion in followup PR that will not be backported?

(BTW, currently including `solana-runtime` will cause circular dependency)

Reply:
Yeah I agree that creating a new migration crate could be a better way to do this but I would prefer to keep the diff small in this PR because we're intending to backport it afaik

Reply:
So you would rather merge this, backport it, and then do the separate crate on `master`?

Reply:
Yes, that's my current thinking. What do you think? My main concern over these static structs getting out of sync is for `master` development.

Reply:
Yeah I think that's fine, as long as we have `master` covered I'm ok with it. I don't think we can backport a new crate (at least not easily) anyways.

Reply:
https://github.com/anza-xyz/agave/pull/3790